rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -163,16 +163,17 @@ bool gui_application::Init()
 
 		bool use_dark_theme = false;
 
-		connect(welcome, &QDialog::accepted, this, [&]()
+		connect(welcome, &QDialog::finished, this, [&]()
 		{
-			use_dark_theme = welcome->does_user_want_dark_theme();
+			use_dark_theme = welcome->use_dark_theme();
 		});
 
+		// welcome dialog is auto-flagged (in its constructor) as WA_DeleteOnClose, so its pointer is no more usable after welcome->exec()
```

Comment:
While it's not necessarily wrong, I don't see the point in adding this comment

Reply:
Of course the code is clear to the main developers, but I usually always provide comments just to make clear for others (e.g. the need to use local variable `bool use_dark_theme` in `gui_application.cpp` that otherwise would be completely useless)

Reply:
It actually doesn't really make sense to use the local variable in the first place.
Just move the guisettings call into the connect